src/flamenco/runtime/program/fd_bpf_loader_v3_program.c

```diff
@@ -175,6 +175,42 @@ fd_bpf_loader_v3_user_execute( fd_exec_instr_ctx_t ctx ) {
     return FD_EXECUTOR_INSTR_ERR_INVALID_ACC_DATA;
   }
 
+  /* Lookup account and its programdata account, error out if program data slot
+    is same as the current slot */
+  FD_BORROWED_ACCOUNT_DECL( program_account );
+  const fd_pubkey_t * program_id_pubkey = &ctx.instr->program_id_pubkey;
+  int err = fd_acc_mgr_view( ctx.txn_ctx->acc_mgr, ctx.txn_ctx->funk_txn, program_id_pubkey, program_account );
```

Comment:
You cannot use acc_mgr in the native programs

Reply:
updating in rewrite of bpf loader